egs2/ms_indic_18/asr1/conf/tuning/train_asr_xlsr53_conformer.yaml

```diff
@@ -0,0 +1,116 @@
+# network architecture
+# encoder related
+# encoder: transformer
+# encoder_conf:
+#     input_layer: "conv2d"
+#     num_blocks: 12
+#     linear_units: 2048
+#     dropout_rate: 0.1
+#     output_size: 256  # dimension of attention
+#     attention_heads: 4
+#     attention_dropout_rate: 0.0
```

Comment:
Can you remove these lines?
If you have specific reasons (some useful information) to keep them, please leave the comments.

Reply:
Yes, I will remove them